.github/workflows/ci.yml

```diff
@@ -68,25 +68,11 @@ jobs:
         run: |
           cmake --build build/ --target coverage
       - name: Check Coverage
-        env:
-          MIN_LINE_COVERAGE: 90
-          MIN_BRANCH_COVERAGE: 70
-        run: |
-          lcov --rc lcov_branch_coverage=1 --list build/coverage.info | tee cov_summary.txt
-          LINE_COVERAGE=$(cat cov_summary.txt | tail -n 1 | cut -d '|' -f 2 | sed -n "s/\([^%]*\)%.*/\1/p")
-          BRANCH_COVERAGE=$(cat cov_summary.txt | tail -n 1 | cut -d '|' -f 4 | sed -n "s/\([^%]*\)%.*/\1/p")
-          RESULT=0
-          echo "Required line coverage: $MIN_LINE_COVERAGE%, actual: $LINE_COVERAGE%"
-          if [[ $(echo "$LINE_COVERAGE < $MIN_LINE_COVERAGE" | bc) -ne 0 ]]; then
-            echo "Line Coverage is too low."
-            RESULT=1
-          fi
-          echo "Required branch coverage: $MIN_BRANCH_COVERAGE%, actual: $BRANCH_COVERAGE%"
-          if [[ $(echo "$BRANCH_COVERAGE < $MIN_BRANCH_COVERAGE" | bc) -ne 0 ]]; then
-            echo "Branch Coverage is too low."
-            RESULT=1
-          fi
-          exit $RESULT
+        uses: FreeRTOS/CI-CD-Github-Actions/coverage-cop@main
+        with:
+          path: ./build/coverage.info
+          line-min-coverage: 90
+          branch-min-coverage: 70
```

Comment:
It shows that the branch coverage is set to 70% but in the actual git actions report when I look at the check coverage test, it says the expected amount is 90% and there is some parsing error. Do you know what's causing this?

Reply:
Good catch! There was a bug in the action's logic which has been fixed in [PR](https://github.com/FreeRTOS/CI-CD-Github-Actions/pull/11)